packages/vue-component/plugin/tag-handler.js

```diff
@@ -160,7 +160,7 @@ VueComponentTagHandler = class VueComponentTagHandler {
       }
 
       // Export
-      script = script.replace(jsExportDefaultReg, 'return');
+      script += `\nreturn require('${this.inputFile.getDisplayPath()}').default;`
```

Comment:
I’m not sure I understand your question. The `require` call only runs client-side (which is counterintuitive, I know).